### DISCUSSION TITLE
每日活跃柱状图的呈现形式

### DISCUSSION THREAD

**Discussion by RicardoDlee** [shion-app/shion#54] [Ideas]
现在在总览部分使用每日活跃柱状图的时候，鼠标移到柱子上，对应的色块会高亮显示，但详细数据框依然会显示所有类目的时长，在色块较多且较接近的时候，会有些不容易区分，能否和色块一样，当鼠标移到对应色块的时候，色块高亮+框内展示对应程序的时长，而当鼠标在柱子区域但在具体色块之外的时候，框内展示对应时段所有程序的时长

**Comment by hanaTsuk1** [comment]
加入开发计划
